__tests__/embed.ts

```diff
@@ -362,7 +362,7 @@ describe('embed.serlo.org/thumbnail?url=...', () => {
       givenWikimedia(defaultWikimediaServer())
     })
 
-    test('returns thumbnail', async () => {
+    test.skip('returns thumbnail', async () => {
```

Comment:
Why do we have to skip it?

Reply:
> Why do we have to skip it?

Reply:
It's for https://github.com/serlo/cloudflare-worker/issues/241. But a good point: I changed that this code is only executed locally and not in e2e tests.